namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-submit-information-report-dev/resources/variables.tf

```diff
@@ -1,3 +1,7 @@
+variable "domain" {
+  default = "hmpps-submit-information-report-dev.service.justice.gov.uk"
```

Comment:
I'd suggest creating `hmpps-submit-information-report.service.justice.gov.uk` instead.
You'll be able to create `dev.hmpps-submit-information-report.service.justice.gov.uk` with an ingress resource later.

Reply:
We've been going round and round with this idea internally. It has been requested that dev is appended as shown. I will raise it again